feat: phone book

Sakuraxx opened the pull request:
Finished all basic requirements of the phonebook task.

VocalNight reviewed (changes requested):
Hey, everything is looking good, but there's 2 things that need tweaking.

- I saw you are using a library to handle the email, but the problem is that it allows me to insert an email without the '.com'

![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Phonebook/assets/12613508/3ebe8c91-31b9-4ba6-a737-10939e372aa5)

See if you can find a way to make it abide to the "xxx@xxx.com" validation. I recommendo you check regex,  https://www.regexone.com/

- You made the project with SQLite, but one of the requirements was to use SQL Server.
- See if you could make that change, you already have most of the work done, so the change should be very simple.

Sakuraxx commented:
> Hey, everything is looking good, but there's 2 things that need tweaking.
> 
> * I saw you are using a library to handle the email, but the problem is that it allows me to insert an email without the '.com'
> 
> ![image](https://private-user-images.githubusercontent.com/12613508/322168879-3ebe8c91-31b9-4ba6-a737-10939e372aa5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTMwOTIxNDEsIm5iZiI6MTcxMzA5MTg0MSwicGF0aCI6Ii8xMjYxMzUwOC8zMjIxNjg4NzktM2ViZThjOTEtMzFiOS00YmE2LWE3MzctMTA5MzllMzcyYWE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDE0VDEwNTA0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc3MTY4NjRlNmM2M2EzNzEzMTAzOTllOTQ2YzM4YTBiZjk5N2I5NTY3NTQ0NjNiYWQwYjNkNzYwMGJmOGQ4MWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ti4Ps8xrEEF-rqFf-gzN3MXF8e3okRFUTiKNLYNmbsw)
> 
> See if you can find a way to make it abide to the "[xxx@xxx.com](mailto:xxx@xxx.com)" validation. I recommendo you check regex, https://www.regexone.com/
> 
> * You made the project with SQLite, but one of the requirements was to use SQL Server.
> * See if you could make that change, you already have most of the work done, so the change should be very simple.

Thanks for reviewing! I fixed the email check issues and changed to sql server localdb. Pls check the code. Thx!
![image](https://github.com/TheCSharpAcademy/CodeReviews.Console.Phonebook/assets/28585709/a654eeee-cc00-4890-bd45-42084ba31e64)

VocalNight reviewed (approved):
Perfect use of regex, well done!.
I'll approve it, good luck on your next projects.
